pkg/tasks/tasks.go

```diff
@@ -21,43 +21,60 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// TaskRunner manages lists of tasks. Through the RunAll method tasks are
+// executed, the lists sequentially, each list of tasks concurrently.
 type TaskRunner struct {
-	client *client.Client
-	tasks  []*TaskSpec
+	client    *client.Client
+	taskLists [][]*TaskSpec
 }
 
+// NewTaskRunner returns a task runner. tasks is the first task list that will
```

Comment:
What do you mean by `tasks is the first task`?

Reply:
`tasks` is referring to the function argument. And it is the first task list that gets processed.